lambdas/es/indexer/index.py

```diff
@@ -32,7 +32,7 @@
 ]
 # 10 MB, see https://amzn.to/2xJpngN
 CHUNK_LIMIT_BYTES = 20_000_000
-DOC_LIMIT_BYTES = 2_000
+DOC_LIMIT_BYTES = os.getenv('DOC_LIMIT_BYTES') or 2_000
```

Comment:
I think you `os.getenv('DOC_LIMIT_BYTES', 2_000)`

Reply:
It is safer as written because it is robust to mistakes in the environment such as `DOC_LIMIT_BYTES = ''` or `DOC_LIMITE_BYTES = 0`.

Reply:
Oh wait... Won't `getenv` give you a string?

Reply:
Oh wait... Won't getenv give you a string?